src/Layer/LabelLayer.js

```diff
@@ -86,7 +101,13 @@ class LabelLayer extends Layer {
 
                 const geometryField = g.properties.style && g.properties.style.text.field;
                 if (!geometryField && !featureField && !layerField) {
-                    return;
+                    // Check if there is an icon, with no text
```

Comment:
could you remove the checking in `Label`?

Reply:
Done